src/vaadin-select.html

```diff
@@ -58,9 +58,11 @@
         invalid="[[invalid]]"
         error-message="[[errorMessage]]"
         readonly$="[[readonly]]"
+        helper-text="[[helperText]]"
         theme$="[[theme]]"
       >
       <slot name="prefix" slot="prefix"></slot>
+      <slot name="helper" slot="helper">[[helperText]]</slot>
```

Comment:
Both are needed and it's a bit tricky the reason behind it. As `vaadin-select` needs to support both string and slot APIs, we need to add the `<slot name="helper" slot="helper">`. That makes `vaadin-text-field` always have a content on its own `helper` slot, so if we just pass the string value to text-field it won't show up. 

So, when the string API is being used, the actual text is coming from the slot. At the same time, we would need to also pass it as property, so the `has-helper` attribute to be correctly added (it can be `has-helper` or `has-helper="slotted"`).